kubeflow/trainer/backends/localprocess/backend.py

```
@@ -0,0 +1,258 @@
+                    steps=[
+                        types.Step(name=s.step_name, pod_name=s.step_name, status=s.job.status)
+                        for s in _job.steps
+                    ],
+                )
+            )
+        return result
+
+    def get_job(self, name: str) -> Optional[types.TrainJob]:
+        _job = next((j for j in self.__local_jobs if j.name == name), None)
+        if _job is None:
+            raise ValueError(f"No TrainJob with name {name}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)